### PR TITLE
Fix Checkstyle errors and compilation errors

### PR DESCRIPTION
**Pull request by alyssaongyx**

### CODE REVIEW

File: src/main/java/seedu/address/model/ReadOnlyPartnerCourseCatalogue.java [hunk 1]
```diff
@@ -3,6 +3,9 @@
 import javafx.collections.ObservableList;
 import seedu.address.model.partnercourse.PartnerCourse;
 
+/**
+ * Unmodifiable view of an address book
+ */
```
Comment (dlathyun) [inline src/main/java/seedu/address/model/ReadOnlyPartnerCourseCatalogue.java:8]: Should it be "Unmodifiable view of partner course catalogue" instead?

File: src/main/java/seedu/address/model/ReadOnlyUniversityCatalogue.java [hunk 2]
```diff
@@ -3,6 +3,9 @@
 import javafx.collections.ObservableList;
 import seedu.address.model.university.University;
 
+/**
+ * Unmodifiable view of an address book
```
Comment (dlathyun) [inline src/main/java/seedu/address/model/ReadOnlyUniversityCatalogue.java:7]: University catalogue?

File: src/main/java/seedu/address/model/partnercourse/PartnerCourse.java [hunk 3]
```diff
@@ -30,6 +37,13 @@ public PartnerName getPartnerName() {
         return partnerName;
     }
 
+    /**
+     * Returns true if both PartnerCourses have the same partnerCode.
+     * Assumption: partnerCode is the primary key i.e. there exists
+     * no two distinct PartnerCourses with the same partnerCode.
+     *
+     * @param otherPartnerCourse This is the other local course.
```
Comment (dlathyun) [inline src/main/java/seedu/address/model/partnercourse/PartnerCourse.java:45]: partner*

File: src/main/java/seedu/address/model/partnercourse/UniquePartnerCourseList.java [hunk 4]
```diff
@@ -57,6 +64,10 @@ public void setPartnerCourse(PartnerCourse target, PartnerCourse editedPartnerCo
         internalList.set(index, editedPartnerCourse);
     }
 
+    /**
+     * Removes the equivalent (as per {@code PartnerCourse#equals(Object)})LocalCourse from the list.
```
Comment (dlathyun) [inline src/main/java/seedu/address/model/partnercourse/UniquePartnerCourseList.java:68]: Partner*

File: src/main/java/seedu/address/model/university/UniqueUniversityList.java [hunk 5]
```diff
@@ -1,29 +1,47 @@
...
     public boolean contains(University toCheck) {
         requireNonNull(toCheck);
         return internalList.stream().anyMatch(toCheck::isSameUniversity);
     }
 
+    /**
+     * Removes the equivalent (as per {@code University#equals(Object)})University from the list.
+     * The University must exist in the list.
+     */
     public void remove(LocalCourse toRemove) {
```
Comment (dlathyun) [inline src/main/java/seedu/address/model/university/UniqueUniversityList.java:45]: Should it be University instead?

File: src/main/java/seedu/address/storage/JsonAdaptedUniversity.java
Comment (dlathyun) [inline src/main/java/seedu/address/storage/JsonAdaptedUniversity.java:1]: For toModelType() should the method be from UniversityName class when checking if the university name is valid?

File: src/main/java/seedu/address/storage/JsonSerializableUniversityCatalogue.java
Comment (dlathyun) [inline src/main/java/seedu/address/storage/JsonSerializableUniversityCatalogue.java:1]: For method toModelType(), you might want to take note of the variable names!

**Comment by alyssaongyx** [comment]
I have updated the code to fix the errors.

**Review by dlathyun — APPROVED** [review]
LGTM

**Pull request merged**